OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -327,59 +327,88 @@ void TracerThread::OpenRingBuffersOrRedirectOnExisting(
   }
 }
 
-bool TracerThread::OpenRingBuffersForTracepoint(
-    const char* tracepoint_category, const char* tracepoint_name, const std::vector<int32_t>& cpus,
-    std::vector<int>* tracing_fds, absl::flat_hash_set<uint64_t>* tracepoint_ids,
-    absl::flat_hash_map<int32_t, int>* tracepoint_ring_buffer_fds_per_cpu,
+namespace {
+
+struct TracepointToOpen {
+  TracepointToOpen(const char* tracepoint_category, const char* tracepoint_name,
+                   absl::flat_hash_set<uint64_t>* tracepoint_stream_ids_to_fill)
+      : tracepoint_category{tracepoint_category},
+        tracepoint_name{tracepoint_name},
+        tracepoint_stream_ids_to_fill{tracepoint_stream_ids_to_fill} {}
+
+  const char* const tracepoint_category;
+  const char* const tracepoint_name;
+  absl::flat_hash_set<uint64_t>* const tracepoint_stream_ids_to_fill;
+};
+
+}  // namespace
+
+static bool OpenFileDescriptorsAndRingBuffersForAllTracepointsOrNone(
```

Comment:
I think OrNone is implied here... maybe just OpenFileDescriptorsAndRighBuffersForAllTracepoints?

Reply:
Done.